src/lib/readmeAPIFetch.ts

```diff
@@ -220,7 +220,9 @@ async function handleRes(res: Response, rejectOnJsonError = true) {
   const contentType = res.headers.get('content-type');
   const extension = mime.extension(contentType);
   if (extension === 'json') {
-    const body = await res.json();
+    // TODO: type this better
```

Comment:
Leaving this as `any` is probably fine imo. You aren't going to have full documented types for these responses without utilizing our API SDK or anything like that (and even then the types are nonexistent for API v1).

Reply:
yeah `node-fetch` is remarkably bad about typing responses